docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -82,36 +77,68 @@ Please refer to our link:../spring-cloud-gcp-samples/spring-cloud-gcp-integratio
 
 Google Cloud Pub/Sub allows many subscriptions to be associated to the same topic.
 `PubSubTemplate` allows you to listen to subscriptions via the `subscribe()` method.
-It relies on a `SubscriberFactory` object, whose only task is to generate Google Cloud Pub/Sub
-`Subscriber` objects.
 When listening to a subscription, messages will be pulled from Google Cloud Pub/Sub
-asynchronously, at a certain interval.
+asynchronously and passed to a user provided message handler.
+
+===== Example
+Subscribe to a subscription with a message handler:
+[source,java,indent=0]
+----
+include::../../../../spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-sample/src/main/java/com/example/WebController.java[tag=subscription]
```

Comment:
M-m-m. Have you checked this with the generated docs?
I feel like this is not going to be available when we have the final HTML doc.

Reply:
Yes, I checked. It works as expected.